src/controllers/userController.js

```diff
@@ -26,10 +26,10 @@ const { unauthorizedUserProfile, serverError, phoneExists } = messages;
 const signUp = async (req, res) => {
   try {
     const {
-      firstName, lastName, email, password, phoneNo
+      firstName, lastName, email, password, phoneNo, lineManager
```

Comment:
add lineManager to swagger doc and what is the data type?

Reply:
I've added lineManager to swagger doc